pxr/imaging/hgiGL/CMakeLists.txt

```diff
@@ -1,7 +1,7 @@
 set(PXR_PREFIX pxr/imaging)
 set(PXR_PACKAGE hgiGL)
 
-if (NOT ${PXR_ENABLE_GL_SUPPORT})
+if (NOT ${PXR_ENABLE_GL_SUPPORT} AND NOT PXR_APPLE_EMBEDDED)
```

Comment:
What's the difference between `AND NOT PXR_APPLE_EMBEDDED` and `&& !defined(ARCH_OS_IPHONE)` ? Should probably use `&&` and `!` in general, but I'm wondering why the different macro here & there?

Reply:
PXR_APPLE_EMBEDDED is what we tagged it in the cmake files as, whereas the ARCH_OS_IPHONE was for the CPP files. The EMBEDDED bit was to be clearer in CMake land where configuration is more common, whereas the OS_IPHONE was to keep in line with the TARGET_OS_IPHONE styling that the compilers already provide.

Regarding the style change to symbols, afaik CMake only supports the words not the symbol? at any rate, none of the existing Pixar cmake uses symbols.

Reply:
Gotcha, it was kinda confusing then, and it's kinda confusing now, but doesn't need an update.